### DISCUSSION TITLE
Problemi con wifi extendere della TP-link

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1267] [HUB: Home Assistant]
Ciao a tutti, 

ho una rogna molto fastidiosa che voglio condividere con voi. 
In casa ho dovuto installare un extender della TP-Link, modello TL-WA850RE, in quanto il segnale Wifi non arriva dovunque.
Da quando l'ho installato, alcuni dispositivi (e in particolar modo i broadlink) ogni tanto non riescono a collegarsi, dandomi questo errore:

`[Errno -4000] Network timeout: No response received within 5s`

Non saprei come comportarmi perchè l'SSID e la password sono gli stessi della rete (ho un TIm-hub H388X), e quasi tutti gli altri dispositivi non mi danno questo problema. Ho anche il robot della roomba che fa i capricci ma sto ancora cercando di capire se è l'extender.

Se provo a staccare l'extender e a riavviare HA (installato in modalità CORE via docker), i broadlink si riconnettono senza problemi.

Avete dei suggerimenti da darmi?
Grazie.

**Comment by 50494554524F** [comment]
conviene usare due ssid se non sono dispositivi mesh

**Reply by calcioscacchi** [reply]
questo vorrebbe dire però collegare i dispositivi a rete diverse. Come fa a non essere un problema?

**Reply by 50494554524F** [reply]
non sono reti diverse, l'extender dovrai connetterlo al router principale

**Reply by calcioscacchi** [reply]
si ma io voglio utilizzare l'extender per collegare anche i dispositivi lontani. 
Quindi dovrò associare tali dispositivi al secondo SSID.

Gestire automazioni tra dispositivi che sono collegati a due SSID è fattibile?

**Reply by 50494554524F** [reply]
li colleghi allo ssid più vicino, non devi far niente altro
